const name = 'sss'
try{
  name = 'eee'
}
catch(err){
console.log(err);
}
console.log(name);